hackathon_site/dashboard/frontend/src/pages/Teams/Teams.module.scss

```diff
@@ -0,0 +1,29 @@
+@import "./../../assets/abstracts/mixins";
+@import "./../../assets/abstracts/variables";
+
+.dashboard {
```

Comment:
I don't think your using anything in this file so you can remove all of the styles here (keep the file tho, in case any css is suddenly required)